cowbird/utils.py

```diff
@@ -155,20 +163,58 @@ def islambda(func):
     return isinstance(func, types.LambdaType) and func.__name__ == (lambda: None).__name__  # noqa
 
 
-def get_app_config(container, celery=True):
+def configure_celery(config, config_ini):
+    logger = get_logger(__name__)
```

Comment:
`logger` should be in caps as in the rest of the project

Reply:
`LOGGER` is used when instatiated in global space and `logger` from a function. I agree though that every file should instantiate the logger in global space for uniformity.

Reply:
Ohh, I missed that one, didn't expand the unchanged lines.

Reply:
Is there any reason though to use the local one here, since it is used in many functions with `__name__` referring to the current module?

Reply:
At first I saw that the get_logger was defined in this file and because of that avoid to call it from the global space. However, you're right, there is no reason to avoid it as long as it is used below the declaration.